src/nylo/tokens/struct.py

```diff
@@ -1,24 +1,48 @@
+# This file is a part of nylo
```

Comment:
struct.py should not have the copyright. Only file beggining with __ shall have them.